login-workflow/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Change Log
 
+## v4.0.1 (Unreleased)
+
+### Fixed
+
+-   Fixed Use waves SVG image for dark theme issue ([#94](https://github.com/etn-ccis/blui-angular-workflows/issues/94)).
```

Comment:
This should be something meaningful like:

### Changed

-   Updated wave image to registration success screen ([#94](https://github.com/etn-ccis/blui-angular-workflows/issues/94)).

Reply:
This actually shouldn't be mentioned in the changelog at all right? This is just a change in the demo project not a change in the package...

Reply:
Revoked the changes in the changelog and package file